### PR TITLE
removed default_middleware_stack

### PR DESCRIPTION
**Pull request by DeonFranklin**
Ref: #112 

This is change 1 of 3

Commit from Hyku containing changes 
https://github.com/samvera/hyku/commit/d7fcd3a3fa3dff6bd98f65ae84ac03282a9c7a27
 
This pull request brings in the change to remove default_middleware_stack.

### CODE REVIEW

File: app/models/collection.rb [hunk 1]
```diff
@@ -1,11 +1,21 @@
 # frozen_string_literal: true
 
 # Generated by hyrax:models
+
 class Collection < ActiveFedora::Base
   include ::Hyrax::CollectionBehavior
-  include Ubiquity::UniversalMetadata
```
Comment (summer-cook) [inline app/models/collection.rb:6]: what is the reason for removing these two lines? is the collection metadata from Ubiquity no longer needed?

**Review by alishaevn — APPROVED** [review]

**Comment by alishaevn** [comment]
is this supposed to be change 1 and 2 of 3?  this pr includes some code from https://github.com/samvera/hyku/pull/1792/commits/d7fcd3a3fa3dff6bd98f65ae84ac03282a9c7a27 _and_ https://github.com/samvera/hyku/pull/1792/commits/dc8f8e6912b444e36bdf78e6e26bb3e165dc0963. it doesn't include all the changes from the second commit though. 

were you planning to add the rest of the code changes from the second commit to this pr?

approving because the code is needed. although the description is fuzzy. 🙃

**Review by jeremyf — CHANGES_REQUESTED** [review]
Waiting on feedback based on @summer-cook's question

**Comment by jeremyf** [comment]
@DeonFranklin My read on this is that you should also be removing `app/actors/hyrax/actors/featured_collection_actor.rb`

File: app/models/collection.rb (hunk 1 above)
Comment (DeonFranklin) [reply app/models/collection.rb:6]: fixed this! thank you!!

**Comment by DeonFranklin** [comment]
featured_collection_actor.rb was removed

**Comment by jeremyf** [comment]
I'm looking at the following code change:

Screen Shot 2023-01-10 at 4 11 13 PM

What I'm seeing is that you removed:

```
include Ubiquity::UniversalMetadata
include Ubiquity::AllModelsVirtualFields
```

Then added back `include Ubiquity::UniversalMetadata` in a different spot.  I'm unclear why one of them came back but not the other.  I'm also unclear why you moved the `include Ubiquity::UniversalMetadata` below `include Hyrax::BasicMetadata`.

The order of includes can matter (e.g. if two modules define the same method, then the module that is included last "wins" in regards to that method definition).

**Comment by jeremyf** [comment]
@DeonFranklin you've now removed `include Ubiquity::UniversalMetadata` and `include Ubiquity::AllModelsVirtualFields`.

Please look at the [Files Changed](https://github.com/scientist-softserv/britishlibrary/pull/239/files) to see what's being removed.

**Comment by DeonFranklin** [comment]
@jeremyf I have identified my source of confusion.

**Comment by jeremyf** [comment]
@DeonFranklin there is still the matter of the order of the includes:

Here's the latest diff.
Screen Shot 2023-01-10 at 7 41 46 PM

**Comment by DeonFranklin** [comment]
@jeremyf I believe this is fixed by this commit:
[51fb81d2be34f86ebb98923e1ee891b729c8c2430](https://github.com/scientist-softserv/britishlibrary/blob/51fb81d2be34f86ebb98923e1ee891b729c8c243/app/models/collection.rb)

**Comment by jeremyf** [comment]
@DeonFranklin click on the [File Changes](https://github.com/scientist-softserv/britishlibrary/pull/239/files)

I'm expecting that the only changes on `app/models/collection.rb` is adding the following:

```ruby
after_update :remove_featured, if: proc { |collection| collection.private? }
after_destroy :remove_featured

def remove_featured
  FeaturedCollection.where(collection_id: id).destroy_all
end
```

But according to the file changes, the modules are in an altered order.

**Pull request closed without merging**